### ISSUE TITLE
Add suffix

### ISSUE THREAD

**Issue by dvirben123** [cchanxzy/react-currency-input-field#99]
any chance to add suffix?

**Comment by cchanxzy** [comment]
Hi, yes that's definitely on the to do list because a few others have asked as well.

I'm in the process of making the component more international friendly, so suffix or at least locale formatting should be coming soon. I don't want to give myself a deadline, but maybe check back in a week or two :-)

**Comment by ianstormtaylor** [comment]
Any update on this? I'd love to be able to use this for percentages as well as currencies, and adding a `%` suffix would be great.

**Comment by cchanxzy** [comment]
Yes there has been some progress on this front.

With v3 release of the component, you can use a suffix with the `intlConfig` option if the locale places the currency symbol after the value e.g. Germany Euro

```js
<CurrencyInput intlConfig={{ locale: 'de-DE', currency: 'EUR' }} />;
```

However, the option to add a custom suffix hasn't been implemented yet. It's probably the next thing on my to do list.

**Comment by cchanxzy** [comment]
Hi, sorry this took a while, but I've finally added the option to specify a custom suffix. 

This is available in [v3.0.2](https://www.npmjs.com/package/react-currency-input-field/v/3.2.0#prefix-and-suffix).

It might be a bit buggy since it's a new feature, so please feel free to open a new issue if you find any bugs.

**Comment by ianstormtaylor** [comment]
@cchanxzy that is awesome, thank you! Loving the component so far btw.
